.gitlab-ci.yml

```diff
@@ -137,6 +137,7 @@ test:backend:
   services:
     - name: postgis/postgis:13-3.0
       alias: postgres
+      command: ["postgres", "-c", "fsync=off"]
```

Comment:
`["postgres", "--fsync=off"]` also works (and "postgres", "-F" but maybe too short.

In my setup I also have `--synchronous_commit=off --full_page_writes=off` but they have probably less impact.